sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -204,17 +204,6 @@ class SparkSession private(
    */
   def listenerManager: ExecutionListenerManager = sessionState.listenerManager
 
-  /**
-   * :: Experimental ::
-   * A collection of methods that are considered experimental, but can be used to hook into
-   * the query planner for advanced functionality.
-   *
-   * @since 2.0.0
-   */
-  @Experimental
-  @Unstable
-  def experimental: ExperimentalMethods = sessionState.experimentalMethods
```

Comment:
I don't think we can actually remove this .. This is being used a lot here and there IIRC ... E.g., see https://community.getorchestra.io/apache-foundation/apache-spark-scala-api-experimentalmethods/

I also used this in my previous workplace too ..

Reply:
By that logic we cannot remove anything ever...

We have a better, more complete, less finicky, alternative.  Let's use that instead.